### ISSUE TITLE
SD_HOST Drivers

### ISSUE THREAD

**Issue by burnbabyburn** [sakaki-/bcmrpi3-kernel-bis#2]
Hi sakaki,

have a look [here ]( https://www.raspberrypi.org/forums/viewtopic.php?t=210225) regarding the following error message on boot:
`[    3.788708] Error: Driver 'sdhost-bcm2835' is already registered, aborting...`

Happens on RPI3 with 64 bit kernel, if default `bcmrp33_defconfig` is used.

**Comment by sakaki-** [comment]
@burnbabyburn,

yes, I see that too in `dmesg`, the upstream drivers are being used it appears. Would make more sense to suppress them and use downstream's version, via the `.config` mods presented in the thread you linked.

I'm busy testing the (forthcoming) 1.3.1 release of [gentoo-on-rpi3-64bit](https://github.com/sakaki-/gentoo-on-rpi3-64bit) right now, but if you'd like to submit a PR for this, I'll review and commit it.

Thanks for drawing my attention to this!

Best, sakaki

**Comment by burnbabyburn** [comment]
Good i already tested and implemented that :)
PLease see pull #3 

It successfully removes the error message.

**Comment by sakaki-** [comment]
I've manually triggered the autobuild, and there's a new release with your changes enabled, [here](https://github.com/sakaki-/bcmrpi3-kernel-bis/releases/tag/4.14.90.20181222).
